src/navigation/sidebar-components/SidebarDropDown.tsx

```diff
@@ -0,0 +1,99 @@
+/*  Name: SidebarDropDown
+    Purpose: Contains drop-down component and view for drop-downs in docs navigation sidebar.
+    Author: M. Solis de Ovando
+*/
+
+import React, { useRef } from "react";
+import styled from "styled-components";
+
+const StyledSpan = styled.span<{ isNestedDropDown: boolean }>`
+  display: block;
+  align-items: center;
+  width: auto !important;
+  font-size: ${(props) => (props.isNestedDropDown ? "14px" : "16px")};
+  line-height: 24px;
+  position: relative !important;
+  padding: 8px 20% 8px 12px !important;
+  cursor: pointer;
+  &:hover {
+    background-color: #e7edee;
+  }
+`;
+
+export const DropDown = (props: {
+  title: string;
+  children?: React.ReactNode;
+  to?: string;
+  isNestedDropDown?: boolean;
+}) => {
+  return (
+    <DropDownView
+      {...useDropDownHook()}
+      title={props.title}
+      isNestedDropDown={props.isNestedDropDown}
+    >
+      {props.children}
+    </DropDownView>
+  );
+};
+
+export function DropDownView(props: DropDownProps) {
+  return (
+    <li className="mm-vertical" ref={props.parentRef}>
+      <StyledSpan
+        isNestedDropDown={props.isNestedDropDown}
+        className="mm-next"
+        // href={id}
+        data-target={props.id}
+        onClick={props.onOpen}
+      >
+        {props.title}
+      </StyledSpan>
+      <div
+        className="mm-panel mm-vertical"
+        id={props.id}
+        style={{
+          marginLeft: "12px",
+          borderLeft: "1px solid #003b45",
+        }}
+      >
+        <ul className="nav-onpage mm-listview mm-vertical">{props.children}</ul>
+      </div>
+    </li>
+  );
+}
+
+export type DropDownProps = {
+  title?: string;
+  onOpen: (e: React.MouseEvent) => unknown;
+  id: string;
+  children?: React.ReactNode;
+  parentRef: React.MutableRefObject<any>;
+  isNestedDropDown?: boolean;
+};
+
+function openVerticalPanel($l) {
```

Comment:
Looks to be identical to `openPanel` from `SidebarCoreCategory`. Try to share a function if possible.